metakb/deltas/civic.py

```diff
@@ -0,0 +1,139 @@
+"""A module for computing CIViC deltas."""
+from metakb import PROJECT_ROOT
+import json
+import logging
+from jsondiff import diff
+from datetime import date
+import pkg_resources
+from metakb.harvesters import CIViC
+
+logger = logging.getLogger('metakb')
+logger.setLevel(logging.DEBUG)
+
+
+class CIVICDelta:
+    """A class for computing CIViC deltas."""
+
+    def __init__(self, main_json, *args, **kwargs):
+        """Initialize the CIVICDelta class.
+
+        :param str main_json: The path to the main CIViC composite json file.
```

Comment:
I think some context about the role played by the `new_json` keyword could be informative here (eg what you said on Slack).

Reply:
Added